terraform/volumes.tf

```diff
@@ -12,3 +12,16 @@
 #  droplet_id = digitalocean_droplet.web.id
 #  volume_id  = digitalocean_volume.minitwit-data.id
 #}
+
+resource "digitalocean_volume" "db" {
```

Comment:
just to understand, this is a pure storage node, and the other resource is the actual computing node?

Reply:
The digitalocean_volume's are just sort of virtual hard drives that can be moved around and expand in size, here's DOs own explanation of them https://docs.digitalocean.com/products/volumes/